automation/capitanodoc/markdown.ts

```diff
@@ -66,7 +66,7 @@ function getAnchor(command: Command) {
 }
 
 export function renderToc(categories: Category[]) {
-	let result = `# Table of contents\n`;
+	let result = `# CLI Command Reference\n`;
```

Comment:
This title actually appears "in the middle" of the [CLI docs web page](https://www.balena.io/docs/reference/cli/). A Table of Contents usually appears at the top of a page... Which is why I thought it would be better to rename it to "CLI Command Reference".